docs/topics/proc_add-debug-logging-nodejs.adoc

```diff
@@ -0,0 +1,36 @@
+= Add Debug Logging
+There are several options for debug logging. This example uses the link:https://www.npmjs.com/package/debug[debug package].
+
+
+.Prerequisites 
+* An application you want to debug. For example, xref:mission-http-api-nodejs[a booster].
+
+.Procedure
+
+. Add the `debug` logging definition.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+var debug = require('debug')('mybooster');
+----
+
+. Add debug statements.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+app.use('/api/greeting', (request, response) => {
+  const name = request.query ? request.query.name : undefined;
+  //log name in debugging
+  debug('name: '+name);
+  response.send({content: `Hello, ${name || 'World'}`});
+});
+----
+
+. Install the link:https://www.npmjs.com/package/debug[debug] module.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install debug
```

Comment:
Would it be better here to show that the `debug` package is present in `package.json`? I don't think you will `npm install` anything in an OpenShift pod, for example.

Reply:
I agree that the `package.json` excerpt is good here. It's also common to show the `install` command. However, it should be shown as `npm install --save debug` which will add the dependency to `package.json` for you.